pkg/apiclient/workflowarchive/workflow-archive.proto

```diff
@@ -10,19 +10,24 @@ package workflowarchive;
 message ListArchivedWorkflowsRequest {
   k8s.io.apimachinery.pkg.apis.meta.v1.ListOptions listOptions = 1;
   string namePrefix = 2;
+  string namespace = 3;
 }
 message GetArchivedWorkflowRequest {
   string uid = 1;
+  string namespace = 2;
 }
 message DeleteArchivedWorkflowRequest {
   string uid = 1;
+  string namespace = 2;
 }
 message ArchivedWorkflowDeletedResponse {
 }
 message ListArchivedWorkflowLabelKeysRequest {
+  string namespace = 1;
 }
 message ListArchivedWorkflowLabelValuesRequest {
   k8s.io.apimachinery.pkg.apis.meta.v1.ListOptions listOptions = 1;
+  string namespace = 2;
 }
```

Comment:
I wasn't absolutely sure whether to add `namespace` to these last two. These seem to be used for populating the drop downs. The requests themselves don't need that information, but I figured if somebody has selected a user namespace this will allow them to get this information.